packages/phone-number-privacy/signer/.env

```diff
@@ -17,4 +17,13 @@ DB_USE_SSL=true
 KEYSTORE_TYPE=MockSecretManager
 KEYSTORE_GOOGLE_PROJECT_ID=mockProjectId
 KEYSTORE_GOOGLE_SECRET_NAME=mockSecretName
-KEYSTORE_GOOGLE_SECRET_VERSION=latest
\ No newline at end of file
+KEYSTORE_GOOGLE_SECRET_VERSION=latest
+# Options: json, human (default), stackdriver
+LOG_FORMAT=stackdriver
```

Comment:
We run our signers in Azure Container Instance and pipe the logs to [Azure Log Analytics](https://portal.azure.com#@7cb7628a-e37c-4afb-8332-2029e418980e/blade/Microsoft_Azure_Monitoring_Logs/LogsBlade/resourceId/%2Fsubscriptions%2F97e2b592-255b-4f92-bce0-127257163c36%2Fresourcegroups%2Fpgpnp-alfajores%2Fproviders%2Fmicrosoft.operationalinsights%2Fworkspaces%2Fpgpnp-alfajores/source/LogsBlade.AnalyticsShareLinkToQuery/q/H4sIAAAAAAAAA3POzytJzMxLLfLMKy5JzEtO9clPj3f2AQCR58T%252FFwAAAA%253D%253D/timespan/P1D). We should double check that this formatting is compatible with [our alerting](https://portal.azure.com/#@mcelo.onmicrosoft.com/resource/subscriptions/97e2b592-255b-4f92-bce0-127257163c36/resourceGroups/pgpnp-alfajores/providers/Microsoft.OperationalInsights/workspaces/pgpnp-alfajores/alerts).

Reply:
Good idea. I'll test on staging and use the 'json' format if not